github/git_commits.go

```diff
@@ -22,6 +22,7 @@ type SignatureVerification struct {
 // Commit represents a GitHub commit.
 type Commit struct {
 	SHA          *string                `json:"sha,omitempty"`
+	HTMLURL      *string                `json:"html_url,omitempty"`
```

Comment:
Move it near `URL`, that should be a more fitting place for it.

See https://github.com/google/go-github/blob/https://github.com/google/go-github/blob/master/github/repos_commits.go#L24/github/repos_commits.go#L24.